oci-artificial-intelligence/ai-speech/customizations/customizations.md

```diff
@@ -103,11 +103,162 @@ To view your newly created customization, click on your customization's name fro
     From either the customization details page or list customizations page, click <strong>delete</strong> to delete your customization
         ![delete customization](./images/delete-customization.png " ")
 
+# Task 4: Using the customizations API
+
+The Postman REST Collection for Customizations setup can be downloaded from [OCI Customization REST Collection](./files/Customization_API_Collection.postman_collection.json). The POSTMAN instructions for the setups are [the same as Speech API](../ai-speech/oci-speech-rest/oci-speech-rest.md)
+
+OCI Speech Service EndPoints for all the services:
+
+*Note:* The list of region identifiers for each region can be found [here](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm), update the endpoints with appropriate region identifiers.
+
+1. <u>Create Customization Job</u>
+    ![Create customization job](./images/customization-rest-post.png)
+
+    Endpoint:
+    ```
+    <copy>
+    https://speech.aiservice.<region-identifier>.oci.oraclecloud.com/20220101/customizations?
+    </copy>
+    ```
+    Body:
+    ```
+    <copy>
+    {
+        "compartmentId": "ocid1.compartment.oc1..<uniqueId>",
+        "description": "<descriptionPlaceholder>",
+        "displayName": "<displayNamePlaceholder>",
+        "alias": "<aliasPlaceHolder>",
+        "modelDetails": {
+            "domain": "GENERIC",
+            "languageCode": "en-US"
+        },
+        "trainingDataset": {
+            "datasetType": "ENTITY_LIST",
+            "referenceExamples": [
+                "Private data is <private>",
+                "Bird name is <bird>"
+            ],
+            "entityList": [{
+                "entityType": "private",
+                "entities": [{
+                        "entityValue": "hello",
+                        "pronunciations": [{
+                                "soundsLike": "helloo"
+                            }
+                        ],
+                        "weight": 1
+                    }
+                ]
+            },{
+                    "entityType": "bird",
+                    "entities": [{
+                            "entityValue": "Tweeteee3",
+                            "pronunciations": [{
+                                    "soundsLike": "tweetyy"
+                                }
+                            ],
+                            "weight": 1
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+    </copy>
+    ```
+    *Note:*
+    * Supported values for domain are GENERIC, and MEDICAL
+    * Supported values for languageCodePLaceHolder are en-US, en-AU, en-IN, en-GB, it-IT, pt-BR, hi-IN, fr-FR, de-DE, es-ES
+
+
+2. <u>Get Customization Job</u>
+    Endpoint:
+    ```
+    <copy> 
+        https://speech.aiservice.us-phoenix-1.oci.oraclecloud.com/customizations/ocid1.aispeechcustomization.dev.dev
```

Comment:
instead of ocid1.aispeechcustomization.dev.dev should just use a simpler placeholder like: <customization_ID> or something, that way we also might not need the clarifying line below